subprojects/docs/src/docs/userguide/command_line_interface.adoc

```diff
@@ -589,7 +589,7 @@ On Linux, OpenJDK's implementation of the file watch service can sometimes miss
 ==== Changes to symbolic links
 
  * Creating or removing symbolic link to files will initiate a build.
- * Modifying the target of a symbolic link will not cause a rebuild.
+ * Modifying the target of a symbolic link (while keeping its content) will not cause a rebuild.
```

Comment:
I think that the added note in brackets is not true: We are monitoring the input directory for changes, so any changes to the target of a symbolic link, if the target is outside of the input directory, will be ignored.

Reply:
K, I'll revert this, it's not critical anyway...

Reply:
Done